wordpress-mobile/WordPress-iOS
# Login prologue: Fixes top constraint for iPhone X

Fixes #7815 

This PR fixes the gap at the top of the login prologue screen on iPhone X. To do this, we now pin the top of the container view in the Login Prologue View Controller to the top of its superview, instead of to the top layout guide.

![login-ix-before-after](https://user-images.githubusercontent.com/4780/30716398-d3090192-9f11-11e7-8ec9-d8a7fcb77660.png)

To test:

* Ensure the login prologue view controller displays correctly on both the iPhone X and other models of iPhone / iPad.

Needs review: @nheagy
